django_cte/raw.py

```diff
@@ -32,7 +32,7 @@ def quote_name_unless_alias(self, name):
     class raw_cte_queryset(object):
         class query(object):
             @staticmethod
-            def get_compiler(connection):
+            def get_compiler(connection, *args, **kwargs):
```

Comment:
I assume this is to allow `elide_empty` to be passed without causing a `TypeError`.

What do you think of changing this to allow/ignore specific parameters rather than having it ignore all? The danger being that it could potentially ignore some future parameter that should not be ignored. In other words, I'd like to keep this interface surface area as small as possible, and have it raise errors for unexpected scenarios.
```suggestion
            def get_compiler(connection, *, elide_empty=None):
```

Reply:
This is a good question. Until now I am not remembering any requests for `using`, so not sure if we need to add it. `elide_empty` is clearly needed because tests fail without it. I think the suggestion is fine for now. We can always add `using` later if needed.